### PR TITLE
Adding a page for Aikar's tunings to PaperDocs

### PR DESCRIPTION
**Pull request by mja00**
I noticed Aikar's blog has been getting quite sluggish lately, which I believe stops a good amount of people from learning what the flags are actually *doing* on their server. I did create a mirror of Aikar's blog post; however, after thinking a bit, I realized it's probably better to be in the docs itself as that'll be more official-looking than some random website that mirrors the content of the page. 

This PR keeps all the same content found on Aikar's blog post, just formatted slightly differently due to how RTD renders. I tried to keep each line in the file under a certain amount of characters, so you aren't horizontally scrolling through the blocks of text when updating/editing anything.

### CODE REVIEW

**Comment by sulu5890** [comment]
Thanks for the PR, it's is a great idea to have this content here. I don't really implementation however of copying the blogpost verbatim. This leads to sections that don't make sense for Paper's docs such as the "Recommended Server Software" section. If this is included here, it also shouldn't be a mirror of the blogpost by linking to it, rather an independent copy of the same material.

Really what I would want is just to have the flags to copy/paste, as well as the breakdown of what each one does but reword it to not sound like a blogpost by removing stuff like "my server" etc. If you've got time to do this that'd be great, else I'm more than happy to do it. Thanks regardless

**Comment by mja00** [comment]
Yeah, I can cut out all the blog post-related stuff and make it more just the needed info. I wasn't sure what was wanted so I just went with verbatim to start. I'll probably get that done sometime tomorrow.

File: source/server/aikar-tuning.rst [hunk 1]
```diff
@@ -0,0 +1,194 @@
+===============
+Aikar's Tuning
+===============
+
+.. contents::
+   :depth: 1
+   :local:
+
+
+The JVM Startup Flags to use – MC 1.15+ (Java 8+, MC 1.8+) Update
```
Comment (sulu5890) [inline source/server/aikar-tuning.rst:10]:
```suggestion
Recommended JVM Startup Flags
```

This section header seems a bit off for a docs article, something like that would be better.

File: source/server/aikar-tuning.rst [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+=================================================================
+
+**Use these flags exactly, only changing Xmx and Xms. These flags work and scale accordingly to any size of memory, even 500MB but 1.15+ will not do well with such low memory…)**
```
Comment (sulu5890) [inline source/server/aikar-tuning.rst:13]:
```suggestion
**Use these flags exactly, only changing Xmx and Xms. These flags work and scale accordingly to any size of memory, even 500MB but modern Minecraft versions will not do well with such low memory…)**
```

I'd like to avoid Minecraft version-specific stuff as much as possible, as it just leads to people asking "Will this work for 1.19 etc."

File: source/server/aikar-tuning.rst [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+   -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M 
+   -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 
+   -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 
+   -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem 
+   -XX:MaxTenuringThreshold=1 -Dusing.aikars.flags=https://mcflags.emc.gs 
+   -Daikars.new.flags=true -jar paperclip.jar nogui
+
+LAST UPDATED: **Apr 25 2020 3:30PM EST**    
+
+These flags are recommended for ALL versions of Minecraft! 1.8 all the way to 1.15+, use this set.
```
Comment (sulu5890) [inline source/server/aikar-tuning.rst:28]:
```suggestion
```
See above, will lead to more questions.

File: source/server/aikar-tuning.rst [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+   It is recommended to reduce your Xmx/Xms by about **1000-1500M** to avoid running out of memory or “OOMKiller” hitting your server. 
+   This also leaves room for the Operating System to use memory too.
+   **Have 8000M memory? Use 6500M for safety.** *But you may also ask your host if they will cover this overhead for you and give you 9500M instead. Some hosts will! Just ask.*
+
+Recommended Memory
+==================
+
+**We recommend using at least 6-10GB**, no matter how few players! If you can’t afford 10GB of memory, give as much as you can, 
+but ensure you leave the operating system some memory too. G1GC operates better with more memory.
+
```
Comment (sulu5890) [inline source/server/aikar-tuning.rst:43]: Something here like "more memory ≠ more performance above a certain point", with the intent of preventing people from going out and buying 24gb from some shared host where that'll give them almost no benefit and just have them paying more.

File: source/server/aikar-tuning.rst [hunk 5]
```diff
@@ -0,0 +1,194 @@
+===============
+Aikar's Tuning
```
Comment (sulu5890) [inline source/server/aikar-tuning.rst:2]:
```suggestion
Aikar's Flags
```

Same for the file name, this is the much more common name and will be more recognizable.

File: source/server/aikar-tuning.rst [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+GC logging does not hurt your performance and can be left on at all times. The files will not take up much space (5MB)
+
+
+Technical Explanation of the Flags
+===================================
+
+#. **-Xms matching -Xmx – Why:** You should never run your server with the case that -Xmx can run the system completely out of memory. 
+   Your server should always be expected to use the entire -Xmx! You should then ensure the OS has extra memory on top of that Xmx for non MC/OS level things. 
+   Therefore, you should never run MC with -Xmx settings you can’t support if java uses it all. Now, that means if -Xms is lower than -Xmx 
+   **YOU HAVE UNUSED MEMORY! Unused memory is wasted memory.** G1 (and probably even CMS to a certain threshold, but we're only stating what we're sure about)
```
Comment (sulu5890) [inline source/server/aikar-tuning.rst:94]: CMS has been removed, not super relevant to mention here anymore (as well as other references)

File: source/server/aikar-tuning.rst [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+=================
+
+Also for Large Pages – It’s even more important to use -Xms = -Xmx! Large Pages needs to have all of the memory specified 
+for it or you could end up without the gains. This memory will not be used by the OS anyways, so use it.
+
+Additionally use these flags (Metaspace is Java 8 Only, don’t use it for Java7): ``-XX:+UseLargePagesInMetaspace``
+
+
+Transparent Huge Pages
+======================
```
Comment (sulu5890) [inline source/server/aikar-tuning.rst:189]: Inconsistent spacing throughout the document; keep one newline here and other places.

**Review by sulu5890 — CHANGES_REQUESTED** [review]
Overall this looks good; just a couple of comments. Also needs a look from Aikar before merging.

**Comment by mja00** [comment]
I'll probably do all of these tomorrow or Thursday, update launch days are always busy for me :P

**Comment by mja00** [comment]
First batch of suggestions done while I had some free time between classes, will get the others done soon™

File: source/server/aikar-tuning.rst [hunk 8]
```diff
@@ -0,0 +1,189 @@
+===============
+Aikar's Flags
+===============
+
+.. contents::
+   :depth: 1
+   :local:
+
+Recommended JVM Startup Flags
+=================================================================
```
Comment (sulu5890) [inline source/server/aikar-tuning.rst:10]:
```suggestion
Recommended JVM Startup Flags
=============================
```

**Review by sulu5890 — APPROVED** [review]

**Comment by sulu5890** [comment]
Need Aikar's OK on this, but, otherwise looks good

**Comment by sulu5890** [comment]
Got Aikar's OK via discord. Thanks.
